spec/01_jukebox_spec.rb

```diff
@@ -70,6 +78,23 @@
     end
```

Comment:
Is there any way we can make the `prints the available commands` test a bit clearer? Those regexes may be hard for students to understand. Maybe the `include` matcher? All we really want to see if the help message includes `help`, `list`, `play` and `exit`.